src/datumaro/util/mask_tools.py

```diff
@@ -7,7 +7,9 @@
 from typing import Dict, List, NamedTuple, NewType, Optional, Sequence, Tuple, TypedDict, Union
 
 import numpy as np
+from pycocotools import mask as pycocotools_mask
```

Comment:
I'm not totally sure why it was inside functions, maybe it should be converted to a lazy import.